models/_sector/nft/trades/solana/nft_solana_trades.sql

```diff
@@ -0,0 +1,83 @@
+{{
+    config(
+        schema = 'nft_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        , materialized = 'view'
+        ,post_hook='{{ expose_spells(\'["solana"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["ilemi"]\') }}'
+    )
+}}
+
+-- ,file_format = 'delta'
+-- ,incremental_strategy = 'merge'
+-- ,partition_by = ['project','block_month']
+-- ,unique_key = ['project','trade_category','outer_instruction_index','inner_instruction_index','account_metadata','tx_id']
+
+
+{% set solana_marketplaces = [
+    ref('magiceden_solana_trades')
+    , ref('tensorswap_solana_trades')
+] %}
+
+
+{% for marketplace in solana_marketplaces %}
+SELECT
+        blockchain,
+        project,
+        version,
+        cast(date_trunc('day', block_time) as date) as block_date,
+        cast(date_trunc('month', block_time) as date) as block_month,
```

Comment:
i notice these fields are also in the project-level spells, we should just pull the column as-is here and avoid building/casting again

Reply:
I'll just cast them here since that was the original ask from Rob